core/http-auth-aws/src/test/java/software/amazon/awssdk/http/auth/aws/internal/signer/DefaultRequestSignerTest.java

```diff
@@ -37,21 +37,20 @@ public class DefaultRequestSignerTest {
                                             .normalizePath(true)
                                             .build();
 
-    V4RequestSigner requestSigner = new DefaultV4RequestSigner(v4Properties);
+    DefaultV4RequestSigner requestSigner = new DefaultV4RequestSigner(v4Properties, "quux");
 
     @Test
     public void requestSigner_sign_shouldReturnSignedContext_butNotAddAnyAuthInfoToRequest() {
```

Comment:
what does "butNotAddAnyAuthInfoToRequest" mean? Don't see any asserts to that effect. Is the idea to assert that `request` before and after is the same? Except the host header? which relates to the other comment I made.

Reply:
Yes, fixed - It asserts that nothing changed on the request.